docs/user-guide/time-and-costs/progress-tracking/README.md

```diff
@@ -10,7 +10,9 @@ keywords: Progress tracking, estimated time, remaining time, work, % complete, p
 
 OpenProject lets you track and monitor the progress of your work packages.
 
-> **Note:** Since OpenProject 14.0, the way progress is reported and calculated has changed significantly. Please read the documentation below to understand how OpenProject handles work and progress estimates.
+> [!IMPORTANT]
+> In OpenProject 14.0, the way progress is reported and calculated has changed significantly. Refer to [this blog article](https://www.openproject.org/blog/changes-progress-work-estimates/) to follow the changes. 
+> **In OpenProject 14.6 some of the changes made in 14.0 were removed and updated**. Please read the documentation below and [this blog post](https://www.openproject.org/blog/updates-to-progress-tracking-in-14-6-based-on-user-feedback/) to understand how OpenProject handles work and progress estimates.
```

Comment:
Given that the changes that made us add the first message were pretty much reverted, I wonder if we can't remove this section. It might be a bit confusing for a user reading the docs, and the history of the changes are not that relevant anymore.

Reply:
@psatyal sometimes the users take a while to update, so I would like to leave this information here for a bit. I can remove it with 15.0 release, hopefully by then the majority of users will have upgraded and will be familiar with the update.